libdevcore/vector_ref.h

```diff
@@ -6,6 +6,10 @@
 #include <vector>
 #include <string>
 
+#ifdef __INTEL_COMPILER
+#pragma warning(disable:597)
```

Comment:
Please could you add a comment to the right of the pragma with the warning message being suppressed?  Thanks!